src/components/ContentUploader/ContentUploader.js

```diff
@@ -148,13 +161,17 @@ class ContentUploader extends Component<Props, State> {
      * @return {void}
      */
     componentWillReceiveProps(nextProps: Props) {
-        const { files, useUploadsManager } = nextProps;
+        const { files, dataTransferItems, useUploadsManager } = nextProps;
+
+        const noFiles = !files || !files.length;
```

Comment:
I would probably use Array.isArray() for readability, since then one knows its an array and not some special object with length attribute.

Reply:
I also think having more +ives is better for readability, something more explicit like below. Uglify js will convert the explicitly written stuff to !s

```js
const hasFiles = Array.isArray(files) && files.length > 0;
const hasItems = Array.isArray(items) && items.length > 0;
const hasUploads = hasFiles || hasItems;

if (!hasUploads) { return; }
```